home/tests/test_insert_question_feature.py

```diff
@@ -110,9 +110,12 @@ def test_post_valid_question_with_client(self, client, valid_data):
                 assert Question.objects.get(title=valid_data["title"])
 
             client.login(username='Lior', password='LiorLior')
-            client.post('/explore/new_question', data=valid_data)
+            response = client.post('/explore/new_question', data=valid_data)
 
```

Comment:
All the tests below seem important.
The question is whether it is possible to split them into several functions because we have received comments in the past about the amount of asserts in that function.

Reply:
All of these asserts are related to the process of posting a valid question:
Once the user has entered a question with valid input, the post should be saved in the database and the user should be automatically navigated to the question page he posted. I can split this test into 2 tests that check if the question exists in the db and if a redirection will be made, but it seems to me that in this case, it is a duplication of code.
Besides, I have previously received a comment from Barak on merging the tests in these cases (in this [PR](https://github.com/beyond-io/shlifim/pull/65)).